vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors.py

```diff
@@ -374,7 +375,8 @@ def _get_scheme_from_parts(
 
         if is_activation_quantization_format(self.quant_format):
             if self._is_fp4a4_nvfp4(weight_quant, input_quant):
-                if CompressedTensorsW4A4Fp4.cutlass_fp4_supported():
+                if CompressedTensorsW4A4Fp4.cutlass_fp4_supported(
```

Comment:
all enviornment variables should be in envs.py and imported with from vllm.envs import ...